src/components/Home.jsx

```diff
@@ -2,23 +2,27 @@ import { useState, useEffect } from 'react'
 import { getReviews } from '../api.js'
 import { ReviewCard } from './ReviewCard.jsx'
 import { formatDate } from '../utils/formatDate.js'
+import { useSearchParams } from 'react-router-dom'
 
 export function Home() {
   const [reviews, setReviews] = useState([])
   const [isLoading, setIsLoading] = useState(false)
 
+  const [searchParams, setSearchParams] = useSearchParams();
+  const categoryQuery = searchParams.get('category')
+
   useEffect(() => {
     setIsLoading(true)
```

Comment:
Does categoryQuery need to be passed down in an array? I have a feeling this might get cumbersome as you asdd more queries here in subsequent tickets.